pulp_puppet_plugins/pulp_puppet/plugins/error_codes.py

```diff
@@ -0,0 +1,8 @@
+from gettext import gettext as _
+
+from pulp.common.error_codes import Error
+
+PUP0001 = Error("PUP0001", _("Could not find module file inside puppet module"), [])
```

Comment:
I believe the file is always called metadata.json so what about s/module file/the 'metadata.json' file/